### PR TITLE
Gettextf fixes and prepare .po files for translators

### PR DESCRIPTION
**Pull request by FransMeerhoff**
Continuation of
https://github.com/jasp-stats/jasp-desktop/pull/4087

- gettextf fixes
- updating .po files
- remove html unicode #4104 
- remove unicode chars from parameter list

### CODE REVIEW

File: JASP-Desktop/JASP-Desktop.pro [hunk 1]
```diff
@@ -1,7 +1,7 @@
 QT      += webengine webchannel svg network printsupport xml qml quick quickwidgets quickcontrols2
 DEFINES += JASP_USES_QT_HERE
 
-GENERATE_LANGUAGE_FILES = false
+GENERATE_LANGUAGE_FILES = true
```
Comment (boutinb) [inline JASP-Desktop/JASP-Desktop.pro]: Should be generate always the language files?

File: JASP-Desktop/widgets/listmodelcustomcontrasts.cpp [hunk 2]
```diff
@@ -378,7 +378,7 @@ void ListModelCustomContrasts::initValues(OptionsTable * bindHere)
 
 	//No need to check colnames to cols in values because they are created during the same loop and thus crash if non-matching somehow
 	if (_values.size() > 0 && int(_values[0].size()) != _rowNames.size())
-		addControlError(tr("Number of rows specifed in Options for ListModelMarginalMeansContrasts does not match number of rows in values!"));
+		addControlError(tr("Number of rows specifed in Options for ListModelCustomContrasts does not match number of rows in values!"));
```
Comment (boutinb) [inline JASP-Desktop/widgets/listmodelcustomcontrasts.cpp]: Change it to Log::log()

**Review by boutinb — CHANGES_REQUESTED** [review]
I wonder why the unicode characters cannot be directly set with their real character: e.g  \u03B8 as θ

**Comment by TimKDJ** [comment]
Does this include all `gettextf()`'s being checked for string literals being passed in the parameters?

**Pull request merged**